cats/templates/cats/explore_list.html

```diff
@@ -0,0 +1,66 @@
+{% extends 'base.html' %}
+{% load crispy_forms_tags %}
+{% block title %}Explore{% endblock %}
+{% block content %}
+    <h1>Explore all Cats</h1>
+    {% if cats_list == None %}
+        <div style="text-align: center; background-color:orange ;margin: auto">
```

Comment:
Like above, use class for properties not defined in bootstrap and bootstrap classes for properties like background color: https://getbootstrap.com/docs/4.0/utilities/colors/#background-color